src/Compiler.js

```diff
@@ -12,6 +12,7 @@ const Statistics = require('./Statistics')
 const CurlyStylesPlugin = require('./plugins/CurlyStylesPlugin')
 const ScopedStylesPlugin = require('./plugins/ScopedStylesPlugin')
 const InternationalizationPlugin = require('./plugins/InternationalizationPlugin')
+const PaddingPlugin = require('./plugins/PaddingPlugin')
```

Comment:
`BoxModelPlugin` https://www.w3.org/TR/CSS2/box.html#mpb-examples

we'll add `border` handling later too